pyproject.toml

```diff
@@ -73,6 +73,7 @@ tox = "^4.4"
 optional = true
 
 [tool.poetry.group.docs.dependencies]
+black = ">=22.10,<24.0"
```

Comment:
black needed for docs as well, if it is missing then there is an INFO message stating that it should be present